frontend/src/components/TradeBox/EncryptedChat/EncryptedSocketChat/index.tsx

```diff
@@ -312,6 +312,12 @@ const EncryptedSocketChat: React.FC<Props> = ({
                 label={t('Type a message')}
                 variant='standard'
                 size='small'
+                multiline
```

Comment:
We might also want to add `maxRows={3}` and allow scrolling. This way the TradeBox does not grow unbounded.